kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/prometheus/PrometheusIntegrationTest.java

```diff
@@ -117,8 +117,11 @@ public void testKieServerStartAndContainersMetrics() {
         assertThat(getMetrics()).contains(
                 "kie_server_start_time{name=\"" + serverName + "\",",
                 "kie_server_deployments_active_total{deployment_id=\"prometheus\",} 1.0",
-                "kie_server_container_started_total{container_id=\"prometheus\",} 1.0",
-                "kie_server_container_running_total{container_id=\"prometheus\",} 1.0"
+                "kie_server_container_running{container_id=\"prometheus\",} 1.0",
+                "kie_server_container_started{container_id=\"prometheus\",} 1.0",
+                "kie_server_container_started_total 2.0",
+                "kie_server_container_running_total 2",
+                "kie_server_container_started_since_boot 2"
```

Comment:
@spolti Can you please remove this line? Or remove value? This metric is affected by other integration tests too.

Reply:
these 4 lines?